src/widgets/TextWidget/index.js

```diff
@@ -40,6 +40,7 @@ class TextWidget extends React.Component {
   static propTypes = {
     theme: PropTypes.shape().isRequired,
     name: PropTypes.string.isRequired,
+    customName: PropTypes.func,
```

Comment:
A variable called `customName` makes no sense here. Please call it `textParser`

Reply:
Done https://github.com/CareLuLu/react-native-web-jsonschema-form/pull/97/commits/659f94fab24a2ee57c5c50e5929d733b0309c88b